librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -312,6 +312,25 @@ commands:
         summary: Debug continue until
         cname: cmd_debug_continue_until
         type: RZ_CMD_DESC_TYPE_OLDINPUT
+  - name: do
+    summary: Debug (re)open commands
+    subcommands:
+      - name: dor
+        summary: Comma separated list of k=v rz-run profile options (e dbg.profile)
+        cname: cmd_debug_process_dor
```

Comment:
try to use names that are more descriptive so that if we have move around commands things still make sense.